### PR TITLE
feat(java):  EKS example

### PR DESCRIPTION
**Pull request by kameshdurga**
Fixes # 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: java/eks/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# CDK Java Example
+
+This is an example of a CDK program written in Java.
+
```
Comment (indrora) [inline java/eks/README.md:4]: This should introduce

* What the sample intends to show
* How to deploy the sample
* What services the sample is interacting with
* How the customer can verify that the deployment is successful

File: java/eks/pom.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        <!-- https://mvnrepository.com/artifact/junit/junit -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.1</version>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
```
Comment (indrora) [inline java/eks/pom.xml:75]: Verify that these are correct for CDKv2

**Review by indrora — DISMISSED** [review]
This needs to be verified for CDKv2 since it's very old.

**Comment by kaizio** [comment]
Stale

**Pull request closed without merging**